docs/pages/Getting_Started.md

```diff
@@ -2306,10 +2307,19 @@ as shown in the last line this need not be the case. The third column must be
 the keyword COMMON_CODE which tells the preprocess program to automatically
 generate all necessary interface code for this type. COMMON_CODE also implicitly 
 means the "model_interpolate()" routine is used as the forward operator.
- 
-:dart: *NOTE*: to add a new quantity, the
+
+<span id="newQuantity" class="anchor"></span> [](#newQuantity)
+### Obs is a new quantity
+To add a new quantity that is not yet defined in DART, the
```

Comment:
'To add a new quantity that is not yet defined in DART' ... is redundant.  It cannot be a new quantity if it is already defined in DART.

Reply:
Michael,

There is more to it than that.  You have to not only add the example as you show, you have to add another line way later in the file. The whole process is described in the obs_def_mod.html ... I just have to figure out how to link to it in stupid github-land.

Reply:
GitHub does not render html ... and it only renders certain .md files.
GitHub takes the docs/pages/*md files and creates GitHub Pages html files, but if you already have html ... you're sunk.

Reply:
> Michael,
> 
> There is more to it than that. You have to not only add the example as you show, you have to add another line way later in the file. The whole process is described in the obs_def_mod.html ... I just have to figure out how to link to it in stupid github-land.

Hey Tim,
You are right, I completely forgot the later initialization line like this
obs_kind_names(367) = obs_kind_type(QTY_ABSOLUTE_HUMIDITY, 'QTY_ABSOLUTE_HUMIDITY')
Shall I close this pull request and open a new one with this additional line, or can this pull request be somehow modified?

Also, have you tried one of those html to markdown converters online? Just a thought.

Michael